app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java

```diff
@@ -57,6 +68,10 @@
 
 public class WalletFragment extends BaseFragment implements SdkStatusListener, WalletBalanceListener {
 
+    private static final String MOONPAY_KEY = "c2tfbGl2ZV9ueVJqVXNDbE5pcnVSdnlCMkJLWW5JcFA5VnA3dWU=";
+    private static final String MOONPAY_URL_FORMAT =
+            "https://buy.moonpay.io?apiKey=pk_live_xNFffrN5NWKy6fu0ggbV8VQIwRieRzy&colorCode=%%232F9176&currencyCode=LBC&showWalletAddressForm=true&walletAddress=%s&externalCustomerId=%s";
```

Comment:
Is it good to have these keys hardcoded like this? I would think these should be build environment variables.